src/query/numeric_opfunc.h

```diff
@@ -40,6 +40,15 @@ typedef enum
   DATA_STATUS_NOT_CONSUMED = 1005	/* Operation not consumed all input */
 } DB_DATA_STATUS;
 
+typedef enum fp_value_type
+{
+  FP_VALUE_TYPE_NUMBER,
+  FP_VALUE_TYPE_INFINITE,
+  FP_VALUE_TYPE_NAN,
+  FP_VALUE_TYPE_ZERO
+}
```

Comment:
indent 수정이 필요할 거 같습니다.
```suggestion
} FP_VALUE_TYPE;
```

Reply:
코드 리뷰 감사합니다.
말씀하신 부분을 반영하여 수정했습니다.

Reply:
FP_VALUE_TYPE은 [67deaff](https://github.com/CUBRID/cubrid/pull/6520/commits/67deaffe99fe84d31321f1536552b012fb539bb1) 에서 phase-1과 phase-2에서만 사용하도록 하고, phase-3에서 원복할 수 있도록 전처리 지시자를 추가했습니다.